src/components/Cards/ExpertCard.js

```diff
@@ -0,0 +1,75 @@
+import Image from 'next/image';
+import styled from 'styled-components';
+
+import {data} from './data/data';
+
+const Card = styled.div`
+	display: flex;
+	flex-direction: row;
+	margin: 0 50px 20px 50px;
+	padding: 15px;
+	border: 1px solid lightgrey;
+	border-radius: 20px;
+`;
+
+const CardImage = styled.div`
+	width: 80px;
+	height: 80px;
+	overflow: hidden;
+	border-radius: 999px;
+`;
+
+const DetailPreview = styled.div`
+	padding: 15px 0 0 30px;
+`;
+
+const CityFont = styled.div`
+	margin-top: 3px;
+	color: lightgrey;
+`;
+
+export default function ExpertCards({searchTerm}) {
+	return (
+		<>
+			<div>
+				{data
+					.filter(item => {
+						return searchTerm.toLowerCase() === ''
+							? item
+							: item.city.toLowerCase().includes(searchTerm);
+					})
+					.map(item => (
+						<>
+							<div>
+								<Card key={item.id}>
+									<CardImage>
+										<Image
+											src="/img/avatar-female.jpg"
+											width={80}
+											height={80}
+											objectFit="cover"
+											alt="portrait"
+										/>
+									</CardImage>
+
+									<DetailPreview>
+										<div>
+											<text>
+												{item.first_name} {item.last_name}
+											</text>
+											<br />
+											<text>{item.ident}</text>
+										</div>
+
+										<div>
+											<CityFont>{item.city}</CityFont>
+										</div>
+									</DetailPreview>
+								</Card>
+							</div>
+						</>
+					))}
+			</div>
+		</>
+	);
```

Comment:
I have some remarks about the semantics and naming of this component:
- First of all! You should work more semantic. :) 
- As your Cards are part of a list you can change the `div` element in row 34/72 to a `ul` element and the `div` element in row 43/69 would be a `li` element. The default styles of a `ul` element that you should reset, if they do not please your styling are `padding-left` and `list-style` (the last one for bullet points). 
- And if I see this correctly, the fragments in row 33/73 and 42/70 are not needed.
- Your `DetailPreview` could also be a `section` instead of a `div`.
- I think `CityFont` is a confusing name as  I thought, this is a special `font-family` you want to use here. Maybe the name `City` would be better.
- `CardImage` is also confusing as this is the `div` that you need to style the nested image inside, but it is not the image itself. I think `ImageWrapper` would be better or `ImageContainer`. :)

Reply:
Ok. Thanks for your feedback. I also changed ExpertCard to ExpertList. It was a note from Ralf.